tests/framework/utils/k8s_helper.go

```diff
@@ -384,6 +363,37 @@ func (k8sh *K8sHelper) WaitUntilPodWithLabelDeleted(label string, namespace stri
 	return false
 }
 
+func (k8sh *K8sHelper) PrintPodStatus(namespace string) {
+	pods, err := k8sh.Clientset.CoreV1().Pods(namespace).List(metav1.ListOptions{})
+	if err != nil {
+		logger.Info("failed to get pod status in namespace %s. %+v", namespace, err)
+		return
+	}
+	for _, pod := range pods.Items {
+		logger.Infof("%s (%s) pod status: %+v", pod.Name, namespace, pod.Status)
+	}
+}
+
+func (k8sh *K8sHelper) PrintPodDescribe(name, namespace string) {
```

Comment:
`kubectl` has a describe command, is that not a standard verb available in the k8s API too?  what does the `kubectl` implementation do?

Reply:
the [kubectl implementation](https://github.com/kubernetes/kubernetes/blob/5782d476c533c1f60c09c09ef940af0f20f59189/pkg/printers/internalversion/describe.go#L205) also gets the object and then prints the events. This isn't quite as elegant, but it was sufficient.